src/Akka.Persistence.PostgreSql/Journal/PostgreSqlJournal.cs

```diff
@@ -45,7 +45,8 @@ public PostgreSqlJournal(Config journalConfig) : base(journalConfig)
                 tagsColumnName: "tags",
                 orderingColumn: "ordering",
                 timeout: config.GetTimeSpan("connection-timeout"),
-                storedAs: storedAs), 
+                storedAs: storedAs,
+                defaultSerializer: "json"), 
```

Comment:
I don't think we should set default to `json` - it's not in line with current defaults if I remember correctly. I.e. Sqlite version uses `config.GetString("serializer")` here.

Reply:
Thanks for the tip of the Sqlite version!